multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/bilibili/BilibiliDto.kt

```diff
@@ -38,6 +38,8 @@ data class BilibiliComicDto(
 @Serializable
 data class BilibiliEpisodeDto(
     val id: Int,
+    @SerialName("is_in_free") val is_in_free: Boolean,
```

Comment:
```suggestion
    @SerialName("is_in_free") val isInFree: Boolean,
```

Change it to camelCase. Update other usages.

Does these appear in the Bilibili Comics API as well? If they don't, it will cause issues and they need to be nullable and have a default value of false.

Reply:
Fix.
Yes, they do appear in the Bilibili Comics, just not useful for checking if the chapter is unlocked.